core/src/main/java/jenkins/telemetry/Telemetry.java

```diff
@@ -141,6 +141,23 @@ public static boolean isDisabled() {
         return jenkins == null || !jenkins.isUsageStatisticsCollected();
     }
 
+    /**
+     * Returns true iff we're in the time period during which this is supposed to collect data.
+     * @return true iff we're in the time period during which this is supposed to collect data
+     *
+     * @since TODO
+     */
+    public boolean isActivePeriod() {
+        LocalDate now = LocalDate.now();
+        if (getStart().isAfter(now)) {
+            return false;
+        }
+        if (getEnd().isBefore(now)) {
+            return false;
+        }
+        return true;
```

Comment:
I at least would find this easier to read as
```suggestion
        return now.isAfter(getStart()) && now.isBefore(getEnd());
```